domains/utkarsh.json

```diff
@@ -0,0 +1,10 @@
+{
+  "repo": "https://github.com/utkarshkrsingh/website",
+  "owner": {
+    "username": "Utkarshkrsingh",
+    "email": "utkarshkrsingh@proton.me"
+  },
+  "record": {
+    "CNAME": "utkarshkrsingh.github.io"
```

Comment:
I except your apology, but I can't seem to access the website.
Gives 404, and can't find a website on your repo either.
if this is the correct record, could you send the URL?

Reply:
My apology,
I am uploaded the files in website repository as well as started the utkarshkrsingh.github.io.
I am really sorry I forgot this and wasted your time. Thank you for informing me.

Reply:
Doing it contribution for first time, that's why a little confused and excited too.